ecr-login/ecr.go

```diff
@@ -52,13 +53,21 @@ func (self ECRHelper) Get(serverURL string) (string, string, error) {
 		return "", "", credentials.NewErrCredentialsNotFound()
 	}
 
+	profile, profile_exists := os.LookupEnv("AWS_PROFILE")
```

Comment:
nit: profile_exists -> profileExists

https://golang.org/doc/effective_go.html#mixed-caps

>  Finally, the convention in Go is to use MixedCaps or mixedCaps rather than underscores to write multiword names.